raiden-dapp/src/components/AddressDisplay.vue

```diff
@@ -29,13 +31,30 @@ export default class AddressDisplay extends Vue {
   copied: boolean = false;
   private timeout: number = 0;
 
+  selectAddress(node: Node): void {
```

Comment:
Interesting, when I was looking for a way to copy text all the articles I could find suggested a hidden text-area. I don't remember seeing this. 

I would guess this works on every modern browser right?

Reply:
Yes worked in Firefox and Chrome, probably in Safari too but Metamask doesnt ;)

It would only require a workaround for IE but we dont need to support it 🎉